docs/current/quickstart/947391-test.mdx

```diff
@@ -58,14 +58,14 @@ The `From()`, `WithDirectory()`, `WithWorkdir()` and `WithExec()` methods all re
 </TabItem>
 <TabItem value="Node.js">
 
-<Embed id="PN7ayA3rjOB" />
+<Embed id="Wb_vszr9Ule" />
 
 This code listing does the following:
 
 - It creates a Dagger client with `connect()` as before.
 - It uses the client's `container().from()` method to initialize a new container from a base image - again, the `node:16-slim` image. This base image is the Node.js version to use for testing. The `from()` method returns a new `Container` object with the result.
-- It uses the `Container.withDirectory()` method to write the host directory to the `/src` path in the container, and the `Container.withWorkdir()` method to set the working directory to that filesystem location.
-  - Notice that the `Container.withDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies) and `ci` (pipeline code) directories.
+- It uses the `Container.withDirectory()` method to mount the source code directory on the host at the `/src` mount point in the container, and the `Container.withWorkdir()` method to set the working directory to that mount point.
+  - Notice that the `Container.WithDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and `build` (build artifacts) directories.
```

Comment:
```suggestion
  - Notice that the `Container.withDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and `build` (build artifacts) directories.
```

Reply:
Thank you! https://github.com/dagger/dagger/pull/6142